src/pages/dashboard/components/dashboardSideMenu/DashboardSidenavItem.tsx

```diff
@@ -0,0 +1,29 @@
+import { ListItemButton, ListItemText, ListItemIcon, Box, Icon } from '@mui/material';
+import { SvgIconComponent } from '@mui/icons-material';
+
+type Props = {
+  handleClick: () => void;
+  title: string;
+  isSelected?: boolean;
+  icon: SvgIconComponent;
+};
+
+export default function DashboardSidenavItem({ handleClick, title, isSelected, icon }: Props) {
+  return (
+    <Box
```

Comment:
is this box required? could we use just the ListItemButton? in this way we have ListItemButton as direct child of List